src/Sobol.jl

```diff
@@ -1,67 +1,93 @@
 module Sobol
 
-import NLopt.libnlopt
-import Base: ndims, length, skip, next, start, done, show, writemime
+import Base: ndims, skip, next, start, done, show, writemime
 export SobolSeq, ScaledSobolSeq, next!
 
-type SobolSeq{N}
-    p::Ptr{Void} # nlopt_sobol pointer
-    function SobolSeq()
-        (N < 1 || N > 1111) && error("invalid Sobol dimension")
-        p = ccall((:nlopt_sobol_create,libnlopt), Ptr{Void}, (Cuint,), N)
-        p == C_NULL && error("error initializing Sobol sequence")
-        s = new(p)
-        finalizer(s, destroy)
-        s
-    end
+include("soboldata.jl") #loads `sobol_a` and `sobol_minit`
+
+type SobolSeq
```

Comment:
I would rather have sdim be a type parameter, i.e. `SobolSeq{sdim}`, since the dimension should be immutable.